README.md

```diff
@@ -120,25 +125,30 @@ $ git clone https://github.com/softdevteam/krun.git
 $ cd krun
 ```
 
-## Step 3: set `JAVA_HOME`
+## Step 3: Build Krun
+
+The Krun Makefile honours the standard variables: `CC`, `CPPFLAGS`, `CFLAGS`
+and `LDFLAGS`. For example, if you wish to use `clang` rather than `gcc` you
+can append `CC=/bin/clang` to the `make` command here:
 
 ```bash
-$ export JAVA_HOME=/usr/lib/jvm/java-7-openjdk-amd64/
+$ pwd
+.../Krun
+$ make
 ```
-## Step 4: Build Krun
 
-The Krun Makefile honours the standard variables: `CC`, `CPPFLAGS`, `CFLAGS`
-and `LDFLAGS`. For example, if you wish to use `clang` rather than `gcc` you
-can append `CC=/bin/clang` to the options here:
+If you want to benchmark Java programs, you will also need to set the
+`JAVA_HOME` environment variable, and build Krun with some extra flags:
 
 ```bash
+$ export JAVA_HOME=/usr/lib/jvm/java-7-openjdk-amd64/
```

Comment:
Similarly for the path here, but I'm torn on this one. The path is obscure and having that concrete example will help users find the Java cruft. Any thoughts?